home/src/main/resources/rdf/display/everytime/menu_de_DE.nt

```diff
@@ -0,0 +1,21 @@
+@prefix display: <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#> .
```

Comment:
In order to avoid a parsing exception, this file must be renamed with `.n3` extension.

Reply:
Ah, ok! Now I got it!